_docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/system/html.html.twig

```diff
@@ -46,64 +46,63 @@
-        "language": "{{ html_attributes['lang'] }}", {# Drupal language code (i.e. $GLOBALS['language']->language) #}
-        "pageName": "{{ head_title['title'] }}".split('|')[0] || "{{ head_title['title'] }}", {# Last section (separated by " | ") of analytics title #}
-        "referringDomain": document.referrer, {# JavaScript: Full referrer URL from document.referrer #}
-        "referringURL": document.referrer.split('/')[2] || "", {# JavaScript: Host portion of document.referrer #}
-        "sysEnv": window.innerWidth <= 768 ? "tablet" : "desktop", {# JavaScript: "tablet" if window.innerWidth <= 768 (pixels), "desktop" otherwise #}
-        "template": "", {# Default empty string for now #}
-        "title": "{{ head_title['title'] }}" {# Metatag title (most of the time matches HTML markup "<title>" value) #}
+  var digitalData = {{ rhd_dtm_script|json_encode(constant('JSON_PRETTY_PRINT'))|raw }};
+
+  ( function( w, d, dd, undefined ) {
+      var categoryObj = getCategories();
+
+      function getCookie( name ) {
+        var value = "; " + document.cookie;
+        var parts = value.split( "; " + name + "=" );
+        if ( parts.length == 2 ) {
+          return parts.pop().split( ";" ).shift();
+        }
+      }
+
+      function getCategories() {
+        var categories = { primaryCategory: "", subCategories: [] },
+          url = window.location.href.split('/');
+        for(var i=3, l=url.length; i<l; i++) {
+          if(i !== 3 && categories.subCategories.length <= 3) { 
+            categories.subCategories.push(url[i]);
+          } else {
+            categories.primaryCategory = url[i];
+          }
+        }
+        return categories;
+      }
+
+      dd.page.category.primaryCategory = categoryObj.primaryCategory;
+      dd.page.category.subCategories = categoryObj.subCategories;
+      dd.page.pageInfo.breadCrumbs = [categoryObj.primaryCategory, categoryObj.subCategories[0] || ""];
+
+      if ( d.referrer ) {
+        var a = d.createElement( "a" );
+        a.href = d.referrer;
+
+        dd.page.pageInfo.referringDomain = a.hostname;
+        dd.page.pageInfo.referringURL = a.href;
       }
-    },
-    "user": [{
-      "profile": [{
-        "profileInfo": {
-          "daysSinceLastPurchase": "", {# Default empty string for now #}
-          "demandbaseSID": "", {# "s_dmdbase" cookie value #}
-          "eloquaGUID": "", {# "rh_elqCustomerGUID" cookie value #}
-          "lastLoginDate": "", {# "true" if "rh_user" cookie value has a value before the first "|", "false" otherwise #}
-          "lastPurchaseDate": "", {# Default empty string for now #}
-          "loggedIn": "", {# Default empty string for now #}
-          "profileID": "", {# Default empty string for now #}
-          "subscriptionFrequency": "", {# Default empty string for now #}
-          "subscriptionLevel": "" {# Default empty string for now #}
+
+      dd.page.pageInfo.sysEnv = ( w.innerWidth <= 768 ) ? "tablet" : "desktop";
```

Comment:
We're not specifying phone as well? Given that we have a lot of mobile traffic for the site, it seems like it would be a good idea.